etc/docker/enterprise-gateway-demo/Dockerfile

```
@@ -1,7 +1,6 @@
 ARG HUB_ORG
-ARG TAG
 
-FROM $HUB_ORG/demo-base:$TAG
+FROM $HUB_ORG/demo-base:2.4.1
```

Per previous comments, we could just reference SPARK_VERSION here...
```suggestion
FROM $HUB_ORG/demo-base:$SPARK_VERSION
```